gbpservice/nfp/orchestrator/drivers/orchestration_driver.py

```diff
@@ -814,13 +879,8 @@ def plug_network_function_device_interfaces(self, device_data,
             raise exceptions.ComputePolicyNotSupported(
                 compute_policy=device_data['service_details']['device_type'])
 
-        token = self._get_token(device_data.get('token'))
-        if not token:
-            return None
-
-        image_name = self._get_image_name(device_data)
-        if image_name:
-            self._update_vendor_data(device_data)
```

Comment:
@ashutosh-mishra Can you comment on this. The update vendor is removed. Is it correct. I thought you added this for a purpose.